mne/viz/evoked.py

```diff
@@ -1350,13 +1350,19 @@ def _parametric_ci(arr, ci=.95):
                        for mean_, sigma_ in zip(mean, sigma)]).T
 
 
-def _bootstrap_ci(arr, ci=.95, n_bootstraps=2000):
+def _bootstrap_ci(arr, ci=.95, n_bootstraps=2000, statfun='mean'):
     """Get confidence intervals from non-parametric bootstrap."""
+    if statfun == "mean":
+        statfun = lambda x: x.mean(axis=0)  # noqa
+    elif statfun == 'median':
+        statfun = lambda x: np.median(x, axis=0)  # noqa
+    elif not callable(statfun):
+        raise ValueError("statfun must be 'mean', 'median' or callable.")
```

Comment:
once I my never-ending welch-reduction PR gets merged you could use the reduction function here - it does what the lines above do.